### PR TITLE
Add full CCD error to the logs

### PR DESCRIPTION
**Pull request by linusnorton**

### CODE REVIEW

**Review by nitinprabhu — APPROVED** [review]

File: src/main/java/uk/gov/hmcts/divorce/systemupdate/service/CcdUpdateService.java [hunk 1]
```diff
@@ -59,6 +59,7 @@ public void submitEvent(final CaseDetails caseDetails,
 
             final String message = format("Submit Event Failed for Case ID: %s, Event ID: %s", caseId, eventId);
             log.info(message, e);
+            log.info(e.contentUTF8());
```
Comment (nitinprabhu) [inline src/main/java/uk/gov/hmcts/divorce/systemupdate/service/CcdUpdateService.java:62]: will it disclose any case data in case of error ?

Comment (linusnorton) [reply src/main/java/uk/gov/hmcts/divorce/systemupdate/service/CcdUpdateService.java:62]: only if CCD does

Comment (nitinprabhu) [reply src/main/java/uk/gov/hmcts/divorce/systemupdate/service/CcdUpdateService.java:62]: safer side we can do only on non prod envs so we don't accidentally leak data ?

Comment (linusnorton) [reply src/main/java/uk/gov/hmcts/divorce/systemupdate/service/CcdUpdateService.java:62]: I would assume anything coming back from their API is going in their logs anyway. I wouldn't be comfortable showing the error to the user but I think putting it in the logs would be okay. I've never seen CCD return case data in an error message

**Pull request merged**